x-pack/plugins/observability/public/embeddable/slo/overview/slo_overview.tsx

```diff
@@ -97,6 +102,10 @@ export function SloOverview({ sloId, sloInstanceId, lastReloadRequestTime }: Emb
       </LoadingContainer>
     );
   }
+
+  if (!isLoading) {
+    onRenderComplete();
+  }
```

Comment:
🤔 I wonder if this is not triggered at every rerender? One way to check would be to console log something here.